api/services/ocean/spark/paths/workspaceStart.yaml

```
@@ -0,0 +1,24 @@
+summary: Ocean Spark
+
+put:
+  summary: "Start Workspace"
+  description: >
+    All Ocean Spark configuration parameters are updatable. This API supports partial updates, so specific fields can be updated separately.
+  operationId: "OceanSparkWorkspaceStart"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../parameters/path/oceanSparkWorkspaceId.yaml"
+    - $ref: "../../../../commons/parameters/accountId.yaml"
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          allOf:
+            - $ref: "../schemas/oceanSparkWorkspaceStartWrapper.yaml"
+  responses:
+    200:
+      description: "Ok"
```

I think the API actually returns the workspace object, so I think we could have :
```suggestion
      $ref: "../responses/workspaces.yaml"
```